application/requirements.php

```diff
@@ -78,13 +78,6 @@
         'memo' => 'Required for PostgreSQL database.',
     ),
     // Cache :
-    array(
-        'name' => 'Memcache extension',
-        'mandatory' => false,
-        'condition' => extension_loaded('memcache') || extension_loaded('memcached'),
-        'by' => '<a href="http://www.yiiframework.com/doc-2.0/yii-caching-memcache.html">MemCache</a>',
-        'memo' => extension_loaded('memcached') ? 'To use memcached set <a href="http://www.yiiframework.com/doc-2.0/yii-caching-memcache.html#$useMemcached-detail">MemCache::useMemcached</a> to <code>true</code>.' : ''
-    ),
```

Comment:
I think this could be left in, unless it caused problems. It's not code we wrote... I think it's just for providing info (if we manually run it) about which of the features that Yii supports we could use, based on whether the feature's requirements are met.